File: backend/Contracts/DF.Contracts/Gateway/Requests/Tracking/LocationRequest.cs
using System;

namespace DF.Contracts.Gateway.Requests.Tracking;

public record CreateLocationRequest(
    string FullAddress,
    string City,
    string Street,
    string House
);
    
public record UpdateLocationRequest(
    string FullAddress,
    string City,
    string Street,
    string House
);
public record AddLocationRequest(
    Guid BusinessId,
    string FullAddress,
    string City,
    string Street,
    string House,
    double Latitude,
    double Longitude
);

File: frontend/food-delivery-platform/src/models/requests/tracking/LocationRequest.ts
export interface CreateLocationRequest {
    fullAddress: string;
    city: string;
    street: string;
    house: string;
}

export interface UpdateLocationRequest {
    fullAddress: string;
    city: string;
    street: string;
    house: string;
}

export interface AddLocationRequest {
    businessId: string; // Guid у C# → string (UUID) у TS
    fullAddress: string;
    city: string;
    street: string;
    house: string;
    latitude: number;
    longitude: number;
}
